Makefile

```diff
@@ -135,7 +135,7 @@ BASE_IMAGES_TAG := $(shell awk '/base_images_tag/ \
 	{split($$NF, a, "\""); print a[2]}' base-images-tag.json)
 
 test-images: docker-images presto-server-rpm.rpm
-	python tests/product/image_builder.py $(IMAGE_NAMES)
+	PYTHONPATH=${PYTHONPATH}:$(shell pwd) python tests/product/image_builder.py $(IMAGE_NAMES)
```

Comment:
if you have a preastoadmin installed in your virtualenv then this change is not needed.

The error you gave points that you don't have a docker installed in your virtualenv. Does this command works without this change? If so then it is strange, or I still need to learn something about python and PYTHONPATH.